src/Microsoft.ML.Data/Transforms/SelectColumnsTransform.cs

```diff
@@ -0,0 +1,632 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.CommandLine;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Transforms;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+[assembly: LoadableClass(SelectColumnsTransform.Summary, typeof(IDataTransform), typeof(SelectColumnsTransform),
+                typeof(SelectColumnsTransform.Arguments), typeof(SignatureDataTransform),
+                SelectColumnsTransform.UserName, "SelectColumns", "SelectColumnsTransform", SelectColumnsTransform.ShortName, DocName = "transform/SelectTransforms.md")]
+
+[assembly: LoadableClass(SelectColumnsTransform.Summary, typeof(IDataView), typeof(SelectColumnsTransform), null, typeof(SignatureLoadDataTransform),
```

Comment:
>SignatureLoadDataTransform [](start = 121, length = 26)

This one is legacy and not needed for a new transformer

Reply:
I have Load and Save tests that fail if this is not defined. Are the tests validating the correct scenario then? Please see TestSelectSavingAndLoading and TestSelectSavingAndLoadingWithNoKeepHidden

---
In reply to: [225387055](https://github.com/dotnet/machinelearning/pull/1269#discussion_r225387055) [](ancestors = 225387055)

Reply:
Sorry, I misread. I thought it was `SignatureDataTransform`. It's good

---
In reply to: [225680465](https://github.com/dotnet/machinelearning/pull/1269#discussion_r225680465) [](ancestors = 225680465,225387055)